main.go

```diff
@@ -115,6 +116,39 @@ type ObjRef struct {
 	Offloaded uint
 }
 
+// updateAutoretrieveIndex ticks every tickInterval and checks for new information to add to autoretrieve
+// If so, it updates the filecoin index with the new CIDs, saying they are present on autoretrieve
+// With that, clients using bitswap can query autoretrieve servers using bitswap and get data from estuary
+func (s *Server) updateAutoretrieveIndex(tickInterval time.Duration, quit chan struct{}) error {
```

Comment:
can we change the name to something more related to the aim of the function? something along the line of `updateIndexer` - I believe this is the end goal.

Reply:
I like having `autoretrieve` on the name so we know that's related to autoretrieve